searx/templates/simple/base.html

```diff
@@ -17,6 +17,9 @@
   {% else %}
   <link rel="stylesheet" href="{{ url_for('static', filename='css/searxng.min.css') }}" type="text/css" media="screen" />
   {% endif %}
+  {% if get_setting('brand.public_instances') %}
+  <link rel="stylesheet" href="/limiter{{ limiter_token }}.css" type="text/css" media="screen" />
+  {% endif %}
```

Comment:
@unixfox: here the limiter is bounded to the UI (I added a random token to the URL).